source/postprocess/tracers.cc

```diff
@@ -262,18 +262,19 @@ namespace aspect
                                                   "remove and add particles|repartition"),
                              "Strategy that is used to balance the computational"
                              "load across processors for adaptive meshes.");
-          prm.declare_entry ("Minimum tracers per cell", "100",
+          prm.declare_entry ("Minimum tracers per cell", "0",
                              Patterns::Integer (0),
-                             "Limit for how many particles are allowed per cell. This limit is "
-                             "useful to prevent coarse cells in adaptive meshes from slowing down "
-                             "the whole model. It will only be checked and enforced during "
-                             "mesh refinement and MPI transfer of tracers.");
+                             "Lower limit for particle number per cell. This limit is "
+                             "useful for adaptive meshes to prevent fine cells from being empty "
+                             "of particles. It will be checked and enforced after mesh "
+                             "refinement and after particle movement.");
```

Comment:
Can you add what happens during enforcement (with possible links to what the various property initializers do)? Same below for the maximal number of particles.